src/java.base/share/classes/java/lang/foreign/Arena.java

```diff
@@ -256,14 +259,14 @@ static Arena ofShared() {
      *     S1.asOverlappingSlice(S2).isEmpty() == true
      * }
      *
-     * @param byteSize the size (in bytes) of the off-heap memory block backing the native memory segment.
+     * @param byteSize the size (in bytes) of the off-heap region of memory backing the native memory segment.
      * @param byteAlignment the alignment constraint (in bytes) of the off-heap region of memory backing the native memory segment.
      * @return a new native memory segment.
-     * @throws IllegalArgumentException if {@code bytesSize < 0}, {@code alignmentBytes <= 0}, or if {@code alignmentBytes}
+     * @throws IllegalArgumentException if {@code bytesSize < 0}, {@code byteAlignment <= 0}, or if {@code byteAlignment}
      * is not a power of 2.
      * @throws IllegalStateException if this arena has already been {@linkplain #close() closed}.
      * @throws WrongThreadException if this arena is confined, and this method is called from a thread {@code T}
-     * other than the arena owner thread.
+     * other than the arena's owner thread.
```

Comment:
"thread T" hints that "T" will be used later, maybe it's not needed.

Reply:
Correct - this is mostly a copy/paste issue for similar throws clauses in methods outside arena.